### PR TITLE
Added a FizzBuzz version for haskell

### PR DESCRIPTION
**Pull request by MrHanSan**
A simple FizzBuzz solution for haskell

### CODE REVIEW

File: haskell/fizzbuzz.hs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fizzbuzz :: [String]
+fizzbuzz = [if n `mod` 15 == 0 then "FizzBuzz" else
```
Comment (Prouser123) [inline haskell/fizzbuzz.hs]: ``fizzbuzz = [if n `mod` 15 == 0 then "FizzBuzz" else``
FizzBuzz should show up if the number is a multiple of 5 & 3, not 15.

**Review by jcxldn — CHANGES_REQUESTED** [review]
N/A - Error in review

**Review by jcxldn — APPROVED** [review]
Thanks for contributing! Your contribution will be merged soon.

**Comment by jcxldn** [comment]
My mistake, numbers divisible by 15 will always be divisible by 3 and 5, but how it's done doesn't matter all that much.

**Pull request merged**